posthog/temporal/data_modeling/run_workflow.py

```diff
@@ -293,12 +294,10 @@ async def handle_model_ready(model: ModelNode, team_id: int, queue: asyncio.Queu
     try:
         if model.selected is True:
             team = await database_sync_to_async(Team.objects.get)(id=team_id)
-            workflow_id = temporalio.activity.info().workflow_id
-            workflow_run_id = temporalio.activity.info().workflow_run_id
-
             saved_query = await get_saved_query(team, model.label)
-            job = await start_job_modeling_run(team, workflow_id, workflow_run_id, saved_query)
```

Comment:
I believe the point of starting this in handle model ready was that we'd be able to have a job run row per model run. Not a blocker but noting that we will have to account for this again once we enable multiple ancestor/descendant mat runs